docs/source/users/options/runtime_option.rst

```diff
@@ -0,0 +1,91 @@
+.. _runtime_option:
+
+###############
+Runtime Options
+###############
+
+The runtime section explains the metrics available for benchmarking the runtime performance.
+
+1) Mean
+-------
+
+This is the default option for benchmarking runtime. If no argument is passed to the ``-rt`` flag from the **cli**, then
+the average runtime of *n* iterations would be represented on the frontend. It can however also be set explicitly by passing
+``mean`` argument to the ``-rt`` flag from the **cli**
+
+.. math::
+
+   \overline{r} = \frac{1}{n} \sum_{i=1}^n x_i = \frac{{x_1}+{x_2}....+{x_n}}{n} -(1)
+
+2) Minimum
+----------
+
+This option can be selected by passing ``minimum`` argument to the ``-rt`` flag from the **cli**. This
+will choose the lowest runtime of *n* iterations to be represented on the frontend. 
+
+.. math::
+
+   runtime = \min[{x_1},....,{x_n}]
+
+3) Maximum
+----------
+
+This option can be selected by passing ``maximum`` argument to the ``-rt`` flag from the **cli**. This
+will choose the highest runtime of *n* iterations to be represented on the frontend. 
+
+.. math::
+
+   runtime = \max[{x_1},....,{x_n}]
+
+
+4) First
+--------
+
+This option can be selected by passing ``first`` argument to the ``-rt`` flag from the **cli**. This
+will choose the first runtime of *n* iterations to be represented on the frontend. 
+
+.. math::
+
+   runtime = {x_1}
+
+5) Median
+---------
+
+This option can be selected by passing ``median`` argument to the ``-rt`` flag from the **cli**. This
+will calculate the median runtime of *n* iterations. 
+
+.. math::
+
+   runtime = {x_\frac{n+1}{2}} \vert {2\nmid n}
+   \\else
+   \\runtime = \frac {{x_\frac{n}{2}}+{x_\frac{n+2}{2}}}{2} \vert {2\mid n}
+   \\when
+   \\runtimes = [{x_1} \leq {x_2} \leq ... \leq {x_n}]
+
+6) Harmonic
+-----------
+
+This option can be selected by passing ``harmonic`` argument to the ``-rt`` flag from the **cli**. This
+will calculate the harmonic mean of *n* iterations and display it on the frontend. 
+
+.. math::
+
+   runtime = \frac{n}{ \sum_{i=1}^n \frac{1}{x_i}}
+    = \frac{n}{\frac{1}{x_1}+\frac{1}{x_2}+ ... + \frac{1}{x_n}} \vert 0 \notin [{x_1}, {x_2}, ... ,{x_n}]
```

Comment:
It's only a minor suggestion but I thought it would be nice to have the "given that" symbol more evident, something like this (not sure this is the proper way of doing this, but kind of gives what I had in mind):

```suggestion
    = \frac{n}{\frac{1}{x_1}+\frac{1}{x_2}+ ... + \frac{1}{x_n}} \quad \Big\vert \quad 0 \notin [{x_1}, {x_2}, ... ,{x_n}]
```